BUILDING_ON_LINUX.md

```diff
@@ -34,50 +35,48 @@ _most likely works the same for other Debian-like distros_
 
 ### Manually
 
-1. `sudo pacman -S qt5-base qt5-multimedia qt5-svg qt5-tools gst-plugins-ugly gst-plugins-good boost rapidjson pkgconf openssl cmake`
-1. go into project directory
-1. create build folder `mkdir build && cd build`
+1. Install all of the dependencies using `sudo pacman -S qt5-base qt5-multimedia qt5-svg qt5-tools gst-plugins-ugly gst-plugins-good boost rapidjson pkgconf openssl cmake`
+1. Go into the project directory
+1. Create a build folder and open it (`mkdir build && cd build`)
+1. Use one of the options below to compile it
 
-#### Using QMake
+### Compiling via CMake
```

Comment:
We're not Compiling via CMake - CMake just generates a set of Makefiles for us. Using is imo better phrasing

Same for the rest of the "Compiling via" changes for CMake/QMake

Reply:
fixed